search/includes/classes/class-dashboard.php

```diff
@@ -19,7 +19,7 @@ public function __construct() {
 	public static function admin_footer_text( $text ) {
 		$screen = get_current_screen();
 		if ( false !== strpos( $screen->base, 'elasticpress' ) || 'ep-pointer' === $screen->post_type ) {
-			$text = '<p>Enterprise Search is powered by <a href="https://10up.com/plugins/elasticpress/">10up\'s ElasticPress plugin</a>.</p>';
+			$text = '<p>Enterprise Search is powered by <a href="https://10up.com/plugins/elasticpress/">10up\'s ElasticPress plugin</a> (v' . esc_html( EP_VERSION ) . ').</p>';
```

Comment:
just on the off chance we add some tests, can we 
```suggestion
			$text = '<p>Enterprise Search is powered by <a href="https://10up.com/plugins/elasticpress/">10up\'s ElasticPress plugin</a> (v' . esc_html( constant( 'EP_VERSION' ) ) . ').</p>';
```